Sources/PubNub/Helpers/PAMToken.swift

```diff
@@ -186,12 +102,12 @@ public struct PAMPermission: OptionSet, Codable, Equatable, Hashable {
   public static let write = PAMPermission(rawValue: 1 << 1) // 2
   public static let manage = PAMPermission(rawValue: 1 << 2) // 4
   public static let delete = PAMPermission(rawValue: 1 << 3) // 8
-  public static let create = PAMPermission(rawValue: 1 << 4) // 16
-
-  public static let update: PAMPermission = [PAMPermission.read, PAMPermission.write]
-  public static let crud: PAMPermission = [PAMPermission.create, PAMPermission.update, PAMPermission.delete]
-
-  public static let all: PAMPermission = [PAMPermission.crud, PAMPermission.manage]
+  public static let get = PAMPermission(rawValue: 1 << 5) // 32
+  public static let update = PAMPermission(rawValue: 1 << 6) // 64
```

Comment:
Why shift? Why not?

```
public static let update = PAMPermission(rawValue: 64)
```

Reply:
@kleewho just sticking with style of existing code.